checkov/common/images/graph/image_referencer_provider.py

```diff
@@ -26,16 +24,14 @@ def __init__(self, graph_connector: LibraryGraph,
                      str, _ExtractImagesCallableAlias]):
         self.graph_connector = graph_connector
         self.supported_resource_types = supported_resource_types
-        self.graph_framework = os.environ.get('CHECKOV_GRAPH_FRAMEWORK', 'IGRAPH')
+        self.graph_framework = os.environ.get('CHECKOV_GRAPH_FRAMEWORK', 'NETWORKX')
```

Comment:
maybe change the default to 'RUSTWORKX'?

Reply:
Done